gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -120,27 +142,41 @@ public void start() throws IOException, InterruptedException {
         entry = zipIn.getNextEntry();
       }
     }
-
-    File datasetFolder = new File(gcdFolder, GCD + '/' + projectId);
+    File datasetFolder = new File(gcdFolder, GCD_VERSION + '/' + projectId);
     deleteRecurse(datasetFolder.toPath());
 
-    // TODO: if System.getProperty("os.name").startsWith("Windows") use cmd.exe /c and gcd.cmd
-    Process temp = new ProcessBuilder()
+    ProcessBuilder processBuilder = new ProcessBuilder()
         .redirectErrorStream(true)
-        .directory(new File(gcdFolder, GCD))
-        .redirectOutput(new File("/dev/null"))
-        .command("bash", "gcd.sh", "create", "-d", projectId, projectId)
-        .start();
+        .directory(new File(gcdFolder, GCD_VERSION));
+    if (isWindows()) {
+      processBuilder.command("cmd", "/C", "gcd.cmd", "create", "-p", projectId, projectId);
+      processBuilder.redirectOutput(new File("NULL:"));
+    } else {
+      processBuilder.redirectOutput(new File("/dev/null"));
```

Comment:
I don't think stdout of the create command adds value here (and can only confuse) but I made the stderr visible.